tests/gtest/test_population_Index.cpp

```diff
@@ -18,22 +18,21 @@ namespace
         // the integration test loops over all ages to test the index.
         for (int fleet_ = 0; fleet_ < population.nfleets; fleet_++)
         {
-            if(population.fleets[fleet_]->is_survey){
             int index_yf = year * population.nfleets + fleet_;
             
             // Currently q is not a vector and not changing over years. 
             // When testing time varying q, better to test entire vector.
             // If not possible to test entire vector, test middle or second to last 
             // than earlier years (collapses to mean in early years)
-            expected_index[index_yf] += population.numbers_at_age[i_age_year]*
+            index_expected[index_yf] += population.numbers_at_age[i_age_year]*
                                         population.fleets[fleet_]->q[0]*
                                         population.fleets[fleet_]->selectivity->evaluate(population.ages[age])*
```

Comment:
does this mean the index is only on Jan 1?

Reply:
Yes at the moment there is no capacity for timing in FIMS or index or any other data types.